README.md

```
@@ -1,69 +1,49 @@
-# 🌱 DiverCompass
+#🌊 DivyDot
 
-> 아카데미 여정을 해저 탐험에 비유하여, 러너가 목표를 잊지 않고 지속적으로 리마인드하며 성장 방향을 확인할 수 있도록 돕는 앱입니다.
+> 아카데미 여정을 해저 탐험에 비유하여,  
+> 러너가 목표를 잊지 않고 지속적으로 리마인드하며 성장 방향을 확인할 수 있도록 돕는 앱입니다.
+
+![alt text](docs/assets/C4702E2E-CBF6-4C9B-B914-DA42B2A6E6F1.png)
 
-![alt text](/docs/assets/image.png)
 
 [![Swift](https://img.shields.io/badge/Swift-6.0.3-orange.svg)]()
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)